compiler/rustc_pattern_analysis/src/usefulness.rs

```diff
@@ -102,7 +102,7 @@
 //! [`Constructor::is_covered_by`].
 //!
 //! Note 1: variable bindings (like the `x` in `Some(x)`) match anything, so we treat them as wildcards.
-//! Note 2: this only applies to matcheable values. For example a value of type `Rc<u64>` can't be
+//! Note 2: this only applies to matchable values. For example a value of type `Rc<u64>` can't be
```

Comment:
Didn't you add an exception for this in `typos.toml`? I don't know which is correct but tf there's an exception this should be kept, presumably.

Reply:
reverted